c_src/membrane_element_sdl/cnodeserver.c

```diff
@@ -0,0 +1,225 @@
+#include <arpa/inet.h>
+#include <netinet/in.h>
+#include <stdio.h>
+#include <string.h>
+#include <sys/socket.h>
+#include <unistd.h>
+#ifndef _REENTRANT
+#define _REENTRANT // For some reason __erl_errno is undefined unless _REENTRANT
```

Comment:
You can try defining `_POSIX_C_SOURCE` as we do in shmex. I could test it, but I don't have the patience to sort out deps...